system-apps/system-settings-preferences/webClient/package.json

```diff
@@ -1,10 +1,10 @@
 {
-  "name": "org.zowe.zlux.appmanager.system.settings-preferences",
+  "name": "org.zowe.zlux.ng2desktop.settings",
   "version": "1.0.0",
   "scripts": {
     "start": "webpack --progress --colors --watch",
     "build": "webpack --progress --colors",
-    "i18n": "ng-xi18n -p tsconfig.i18n.json --i18nFormat=xlf  --outFile=messages.xlf",
+    "i18n": "ng-xi18n -p tsconfig.ngx-i18n.json --i18nFormat=xlf  --outFile=messages.xlf && xliffmerge -p xliffmerge.json",
```

Comment:
build fails because xliffmerge doesn't exist, neither does xliffmerge.json
This seems to be for the desktop itself, not apps.

Other apps don't do this:
https://github.com/zowe/sample-angular-app/blob/master/webClient/package.json#L7